packages/neuron-wallet/src/utils/prompt/password/index.ts

```diff
@@ -0,0 +1,44 @@
+const { ipcRenderer } = require('electron')
+
+const form = document.querySelector('form') as any
+
+const { language } = window.navigator
+
+const locale = (() => {
+  if (language === 'en-US' || language === 'en') {
+    return 'en-US'
+  }
+  return 'zh-CN'
+})()
+
+const labels: any = {
```

Comment:
Can we move this to `locales` dir ?

Reply:
> Can we move this to `locales` dir ?

updated in https://github.com/nervosnetwork/neuron/pull/450/commits/3d11cad35ac3d1306abce7d8cfe40476c7a7fff0